packages/docusaurus-theme-classic/src/theme/SkipToContent/styles.module.css

```diff
@@ -6,9 +6,6 @@
   padding: calc(var(--ifm-global-spacing) / 2) var(--ifm-global-spacing);
   color: var(--ifm-color-emphasis-900);
   background-color: var(--ifm-background-surface-color);
-  border-radius: var(--ifm-global-radius);
```

Comment:
This is not directly related to updating Infima, just removing unnecessary styles and adding shadow to the skip link (see above).